Packs/AlphaSOC_Network_Behavior_Analytics/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### AlphaSOC Network Behavior Analytics
+- Adjust the default minimum severity from 3 to 4
```

Comment:
Changed the default value for the *Ignore events below severity* parameter from 3 to 4.